aodncore/pipeline/handlerbase.py

```diff
@@ -138,39 +138,123 @@
 ----------
 
 A handler instance contains a number of attributes which control or modify the behaviour of the handler. The attributes
-are typically set from the **params** key of the watch configuration, or from the ``__init__`` method of a handler
-subclass (e.g. when writing tests).
+are typically set from the **params** key of the watch configuration, or as initialisation parameters to ``__init__``
+method of a handler subclass (e.g. when writing tests).
 
 Class parameters
 ~~~~~~~~~~~~~~~~
 
-The following class parameters are also assigned to attributes of the same name, as a convenience.
+The class parameters are also assigned to instance attributes of the same name, as a convenience.
```

Comment:
Why "also"? What else are they assigned to?

Reply:
Possibly redundant, and related to my other comment, but the also is that the __init__ params are assigned to local variables in the __init__ scope and *also* added as instance attributes.

Reply:
Ok. I don't think the "init scope" is relevant in this doco.